coriolis/osmorphing/osmount/windows.py

```diff
@@ -149,11 +178,16 @@ def _set_basic_disks_rw_mode(self):
             logmsg_fmt="Clearing R/O flag on disk with ID '%s'.",
             skip_on_error=True)
 
-    def _bring_disk_offline(self, drive_letter):
-        self._conn.exec_ps_command(
-            "Get-Volume |? DriveLetter -eq \"%s\" | Get-Partition | "
-            "Get-Disk | Set-Disk -IsOffline $True" % drive_letter,
-            ignore_stdout=True)
+    def _bring_disks_offline(self, drives_to_skip=None):
+        disk_ids_to_skip = None
+        if drives_to_skip:
+            disk_ids_to_skip = self._get_disk_ids_from_drive_letters(
+                drives_to_skip)
+        offline_disk_script_fmt = "SELECT DISK %s\r\nOFFLINE DISK\r\nEXIT"
```

Comment:
Iirc if we have Dynamic Disks, then bringing one of the physical disks which comprise the dynamic disk offline will (rightfully) bring all the others offline.
In this sense, please double-check that running `OFFLINE DISK` on an already offline disk won't lead to an error code.

Reply:
I've tested this with a source VM that has actual spanned dynamic disks, and it didn't error out. Will double-check